packages/utilities/psammead-rich-text-transforms/src/candy-xml.js

```diff
@@ -44,13 +44,25 @@ const createUrlLink = element => {
   return urlLink(text, locator, blocks);
 };
 
+const handleSupportedNodes = (childNode, attributes, acc) => {
+  const block = xmlNodeToBlock(childNode, attributes);
+  const blocks = is(Array, block) ? block : [block];
+  return [...acc, ...blocks];
+};
+
+const handleUnsupportedNodes = (childNode, attributes) =>
```

Comment:
anonymous wrapper needed?  seems this could be `const handleUnsupportedNodes = convertToBlocks`, or just deleted entirely

Reply:
I extracted this into it's own method in the hope the naming would be more clear, thinking about it I should have just added an inline comment